src/services/vc-status-list.service.ts

```diff
@@ -37,15 +37,15 @@ export const getNewVcStatusIndex = async (controllerDID: string) => {
   );
 
   VcStatusIndexControllerModel.createVcStatusIndexController(
-    statusFile._id,
-    statusFile.lastIndexInUse,
+    statusFile!._id,
```

Comment:
@vijayshiyani need to look into how to configure proper typing for these cases.

Reply:
not sure how to fix this.